### ISSUE TITLE
Design PocketHub logo

### ISSUE THREAD

**Issue by fadils** [pockethub/PocketHub#905]
We're in need of a revised logo before publishing it to the Play Store.

Can someone who can draw, give us a hand at this?

@hzsweers @Meisolsson if you guys want to add some characteristics of the logo, please do.

As for me, It doesn't have to be fancy. Simple will be fine.

**Comment by EzraBrooks** [comment]
Octocat poking his head out of a pocket would be a great logo. Are we allowed to use Octocat's likeness?

**Comment by Meisolsson** [comment]
https://octodex.github.com/faq.html looks like a no on the octocat

**Comment by EzraBrooks** [comment]
Bummer. Well, something with a pocket, definitely.

**Comment by larsgrefer** [comment]
Take a look at #771

**Comment by Meisolsson** [comment]
What about having the android robot peeking out of a pocket and holding the git logo? (Only the lines/dots part)

**Comment by larsgrefer** [comment]
sounds nice

what about the robot having tentacles instead of legs?

**Comment by hzsweers** [comment]
I like the idea of a repo icon in a pocket. Octo and tentacles would either
be too close to the GitHub icon for legal reasons or not close enough to be
obvious. Just my opinion. Repo icon is obvious.

**Comment by Meisolsson** [comment]
Well is the repo icon not also from a octocat?

**Comment by benklop** [comment]
The lines/dots icon is from the git project, not github. its licensing is covered here: http://www.git-scm.com/trademark - I'd say that it could reasonably be used without permission "To factually refer to the Git Project itself, its products, or its protocols", though since this is for GitHub, and not just Git, and wouldn't work with a vanilla Git repo, it's possible this would fall under 'substantially modified' and not be allowed.

**Comment by larsgrefer** [comment]
This issue was fixed in #951 and should be closed

**Comment by Meisolsson** [comment]
Not really, the new logo is temporary until we find a better alternative

**Comment by clomads** [comment]
Still looking at options for icons? I'll ponder some potential directions.

**Comment by DataSensible** [comment]
Can I call a quorum for smart, and good people to discuss humanity? Is it a problem  to include nationals? Yes but it is the correct way to sample and begin new habitat for humanity projects. Sapiophiles where ya at? Detente and open lines of communication in event of crisis, we can find a viable option together or I won't by myself.
